src/main/java/com/github/manolo8/darkbot/core/entities/BattleStation.java

```diff
@@ -75,8 +76,7 @@ public boolean isRemoved() {
 
     @Override
     public boolean use() {
-        boolean allowEnemy = main.hero.invisible && main.config.GENERAL.ROAMING.ENEMY_CBS_INVISIBLE;
-        return hullId > 0 && hullId < 255 && (info.isEnemy() && !allowEnemy);
+        return false;
```

Comment:
if use is hard-coded to false, why even implement obstacle?

Reply:
to do not break plugins, also in API battlestation implement Obstacle